satviz-consumer/src/main/java/edu/kit/satviz/consumer/gui/config/GeneralConfigController.java

```diff
@@ -285,10 +289,10 @@ protected void loadConsumerConfig(ConsumerConfig config) {
 
     windowSizeSpinner.getValueFactory().setValue(config.getWindowSize());
 
-    HeatmapColors colors = config.getHeatmapColors();
-    if (colors != null) {
-      coldColorColorPicker.setValue(colors.getColdColor());
-      hotColorColorPicker.setValue(colors.getHotColor());
+    Theme theme = config.getTheme();
+    if (theme != null) {
```

Comment:
Hier sollte vermutlich sicherheitshalber noch überprüft werden, dass die einzelnen Colors nicht `null` sind
```suggestion
    if (theme != null && theme.getColdColor() != null && theme.getHotColor() != null) {
```

Reply:
Merci. J'ai fais ca.